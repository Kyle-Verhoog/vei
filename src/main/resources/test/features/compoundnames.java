public class Compoundnames {
  public Compoundnames() {}
  public void m() {
    java.util.Vector x = null;
  }
}