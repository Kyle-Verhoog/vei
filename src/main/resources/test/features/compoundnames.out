PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
PUBLIC ┃ public
VOID ┃ void
IDENTIFIER ┃ m
( ┃ (
) ┃ )
{ ┃ {
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ util
. ┃ .
IDENTIFIER ┃ Vector
IDENTIFIER ┃ x
= ┃ =
NULL_LITERAL ┃ null
; ┃ ;
} ┃ }
} ┃ }